css/index.css

```diff
@@ -112,6 +252,46 @@ p {
     justify-content: center;
 }
 
```

Comment:
good job targeting the boxes individually. you could also have added a secondary class in place of the id